cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -151,6 +151,15 @@ function _add_common_params() {
         params=" --docker-proxy=$KUBEVIRTCI_PROXY $params"
     fi
 
+    # MULTI_NUMA for now means 2 NUMA nodes, it assumes the memory can be divided by 2 and cpu count is 6
```

Comment:
For consistency and for simplicity(at least from my view) we should move this to vm.sh/run.go. Here should be just a simple --numa flag that enables the configuration.

Reply:
so we would just have one env var `KUBEVIRT_NUMA_NODES` which will default to `1`? And the logic to create proper `qemu` args can be moved to `go` code. Sounds good!

Reply:
Yes. This seems to me more intuitive and less complex.

Reply:
I have moved the logic to `run.go`